src/js/tabs/pid_tuning.js

```diff
@@ -1715,13 +1716,26 @@ pid_tuning.initialize = function (callback) {
                 };
             }
 
+            function calculateThrottlePosition(thrPercent, canvasWidth, canvasHeight, mid, midx, midxl, midxr, midy, midyr, midyl) {
+                return thrPercent <= mid ? getQuadraticCurvePoint(0, canvasHeight, midxl, midyl, midx, midy, thrPercent * (1.0 / mid)) : getQuadraticCurvePoint(midx, midy, midxr, midyr, canvasWidth, 0, (thrPercent - mid) * (1.0 / (1.0 - mid)));
```

Comment:
Just to maintain the style of the ternary:

```suggestion
                return thrPercent <= mid
                    ? getQuadraticCurvePoint(0, canvasHeight, midxl, midyl, midx, midy, thrPercent * (1.0 / mid))
                    : getQuadraticCurvePoint(midx, midy, midxr, midyr, canvasWidth, 0, (thrPercent - mid) * (1.0 / (1.0 - mid)));
```

Reply:
I don't see the benefit. The current line is long, but not too much. I prefer the ternary in one line.